pkg/reconciler/eventpolicy/eventpolicy.go

```diff
@@ -28,28 +28,28 @@ import (
 )
 
 type Reconciler struct {
-	fromRefResolver *resolver.AuthenticatableResolver
+	authResolver *resolver.AuthenticatableResolver
 }
 
 // ReconcileKind implements Interface.ReconcileKind.
 // 1. Verify the Reference exists.
 func (r *Reconciler) ReconcileKind(ctx context.Context, ep *v1alpha1.EventPolicy) pkgreconciler.Event {
 	featureFlags := feature.FromContext(ctx)
 	if featureFlags.IsOIDCAuthentication() {
-		ep.GetConditionSet().Manage(ep.GetStatus()).MarkTrue(v1alpha1.EventPolicyConditionAuthnEnabled)
+		ep.Status.MarkOIDCAuthenticationEnabled()
 	} else {
-		ep.GetConditionSet().Manage(ep.GetStatus()).MarkFalse(v1alpha1.EventPolicyConditionAuthnEnabled, "AuthOIDCFeatureNotEnabled", "")
+		ep.Status.MarkOIDCAuthenticationNotEnabled("AuthOIDCFeatureNotEnabled", "")
```

Comment:
```suggestion
		ep.Status.MarkOIDCAuthenticationNotEnabled("OIDCAuthenticationDisabled", "")
```

Reply:
@creydr I think I should rename `MarkOIDCAuthenticationNotEnabled` to `MarkOIDCAuthenticationDisabled` as well to keep it consistent...no?

Reply:
Good point :+1: